src/components/media/MediaFigure.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+import './Media.scss';
+
+type Props = {
```

Comment:
Can you convert all the `Props` in these files to exported interfaces with specific names, i.e., `export interface MediaFigureProps`?